website/docs/cluster-management/managing-clusters-without-capi.mdx

```diff
@@ -12,9 +12,8 @@ import BrowserOnly from "@docusaurus/BrowserOnly";
 
 # Managing Clusters Without Cluster API <TierLabel tiers="Enterprise" />
 
-You do **not** need Cluster API to add your Kubernetes cluster to Weave Gitops Enterprise. The only thing you need is a secret containing a valid `kubeconfig`.
+You do **not** need Cluster API to add your Kubernetes cluster to Weave GitOps Enterprise. The only thing you need is a secret containing a valid `kubeconfig`.
 
-import TOCInline from "@theme/TOCInline";
```

Comment:
this shouldn't be removed I think or line 18 should also be removed with this
![Screenshot 2023-08-18 at 13 12 51](https://github.com/weaveworks/weave-gitops/assets/10995114/1f885016-7ca4-442c-a230-cf5c1381166d)

Reply:
yeah if line 17 changes were not intentional then we should add it back.

Reply:
done